packages/lib/isOutOfBounds.tsx

```diff
@@ -250,6 +250,15 @@ export function isTimeViolatingFutureLimit({
       isAfterRollingEndDay,
       endOfRollingPeriodEndDayInBookerTz: periodLimits.endOfRollingPeriodEndDayInBookerTz.format(),
     });
+    if (isAfterRollingEndDay)
+      log.warn(
+        "Booking is out of bounds due to rolling period end day.",
+        safeStringify({
+          formattedDate: dateInSystemTz.format(),
+          isAfterRollingEndDay,
+          endOfRollingPeriodEndDayInBookerTz: periodLimits.endOfRollingPeriodEndDayInBookerTz.format(),
+        })
+      );
```

Comment:
We log all necessary details when user attempts to book after the rolling period